diff --git a/Dockerfile-auth b/Dockerfile-auth
index c60020061..df5584a70 100644
--- a/Dockerfile-auth
+++ b/Dockerfile-auth
@@ -9,7 +9,7 @@ ENV NO_LUA_JIT="s390x arm64"
 RUN apt-get update && apt-get -y dist-upgrade && apt-get clean
 
 # devscripts gives us mk-build-deps (and a lot of other stuff)
-RUN apt-get update && apt-get -y dist-upgrade && apt-get install -y  --no-install-recommends devscripts dpkg-dev equivs git python3-venv && apt-get clean
+RUN apt-get update && apt-get -y dist-upgrade && apt-get install -y  --no-install-recommends devscripts dpkg-dev equivs git liblua5.3-dev python3-venv && apt-get clean
 
 # import everything - this could be pdns.git OR an auth tarball!
 COPY builder-support /source/builder-support
